### PR TITLE
Specify exponential ramp behavior when previous value is 0

### PR DESCRIPTION
**Pull request by rtoy**
Clarify what happens if the previous event value is 0.  We also
explicitly allow v0 and v1 to both be negative (the formula is
well-defined in this case).

If an error is thrown, the previous value is propagated.

Fixes WebAudio/web-audio-api#612.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2396,9 +2397,23 @@ <h2 id="AudioParam">
               </dt>
               <dd>
                 The value the parameter will exponentially ramp to at the given
-                time. A NotSupportedError exception MUST be thrown if this
-                value is less than or equal to 0, or if the value at the time
-                of the previous event is less than or equal to 0.
+                time. A NotSupportedError exception MUST be thrown if any
```
Comment (padenot) [inline index.html]: `<code>` around the exception name.

Comment (rtoy) [inline index.html]: Ok.  Are we going to do that everywhere else in the document?  We don't do that anywhere right now.

Comment (padenot) [inline index.html]: Yeah I think I have a patch that does it lying around in a branch.

File: index.html [hunk 2]
```diff
@@ -2376,9 +2376,10 @@ <h2 id="AudioParam">
               - T_0}{T_1 - T_0} $$
             </p>
             <p>
-              where \(V_0\) is the value at the time \(T_0\) and \(V_1\) is the
-              <code>value</code> parameter passed into this method.  It is an error if either
-              \(V_0\) or \(V_1\) is not strictly positive.
+              where \(V_0\) is the value at the time \(T_0\) and \(V_1\) is
+              the <code>value</code> parameter passed into this method.  It
+              is an error if \(V_0\) and \(V_1\) have opposite signs or if
+              either \(V_0\) or \(V_1\) is zero.
```
Comment (rtoy) [inline index.html]: Actually, I have some second thoughts about this.  I think it's useful if V_1 is zero to signal an error.  But if you're running the automation and when the exponential ramp event becomes current and we then discover that V_0 is 0, throwing an error is kind of useless.  You get notified, but there's nothing you can do about it, and the error can happen long after the exponential ramp has already finished.

File: index.html (hunk 1 above)
Comment (rtoy) [inline index.html]: Although I did this in this patch, couldn't we do this the same way that "MUST" is magically highlighted (by respec)?

**Pull request merged**